cmd/bedpeOverlap/bedpeOverlap.go

```diff
@@ -257,12 +261,12 @@ func main() {
 		log.Fatalf("Error: overlap threshold must be between 0 and 1")
 	}
 
-	//should no longer be needed when overlapThresholdBedPe is implemented
-	if *overlapThreshold != 0 && *bedSelect == false {
+	// should no longer be needed when overlapThresholdBedPe is implemented
+	if *overlapThreshold != 0 && *bedSelect {
 		log.Fatalf("Error: overlapThreshold must be used with bedSelect")
 	}
 
-	if *overlapBoth == true && *bedSelect == false {
+	if *overlapBoth && *bedSelect {
```

Comment:
So I think this is going to fatal when both are true, but the fatal condition is overlapBoth==true, bedSelect==false.

Reply:
my bad meant to put `!`